modules/ROOT/pages/to-publish-assets-maven.adoc

```diff
@@ -17,6 +17,8 @@ The Exchange Maven Facade API does not support deleting assets. To delete an ass
 
 The Exchange Maven Facade API does not support dynamic parameters such as setting the asset version to `1.0.${revision}`.
 
+Dynamic parameters for fields other than <name> and <description> are not supported. If the uploaded asset has a defined parent, the parent must be a valid and already uploaded Exchange asset for the dynamic parameters to be properly interpreted.
```

Comment:
Add some backticks to the name and description tags to avoid formatting issues

```suggestion
Dynamic parameters for fields other than `<name>` and `<description>` are not supported. If the uploaded asset has a defined parent, the parent must be a valid and already uploaded Exchange asset for the dynamic parameters to be properly interpreted.
```